static/js/components/ValidationAlert.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+
+import type { ValidationErrors } from '../flow/profileTypes';
+
+export default class ValidationAlert extends React.Component {
+  props: {
+    errors:   ValidationErrors,
+    children: React$Element[],
+  };
+
+  alertMessage: Function = (): React$Element|void => {
+    const { errors } = this.props;
+    if ( !_.isEmpty(errors) ) {
+      return (
+        <span className="message">
+          Please fix the errors above and try again
+        </span>
+      );
+    }
+  };
+
+  render () {
+    const { children } = this.props;
+    return (
+      <div className="validation-alert">
+        { this.alertMessage() }
+        <div className="actions">
+          { children }
+        </div>
+      </div>
```

Comment:
hmmm. i guess this is material-ui forcing us to wrap the buttons in this new component since you can only render content in the bottom bar via the `actions` property. i think it makes sense to create our own component (`ValidatingDialog` or something) that encapsulates these changes. we could leave `actions` alone in all of these files, and in `EducationDialog.js` (just as an example) the API would change to something like this.

```javascript

# In the render method...
const { errors } = this.props;
<ValidatingDialog
           open={workDialogVisibility}
           className="dashboard-dialog employment-dashboard-dialog"
           onRequestClose={this.closeWorkDialog}
           actions={actions}
           autoScrollBodyContent={true}
           errors={errors}
         >
```

`ValidatingDialog` would wrap the actions with `<div className="validation-alert">` and show the message only when validation errors exist. it would then just pass everything along to `Dialog`